package.json

```
@@ -111,6 +116,12 @@
       "sfpowerkit:pool": {
         "description": "Commands to create a pool of scratch orgs"
       },
+      "sfpowerkit:project": {
+        "description": "Commands that operate on the project"
+      },
+      "sfpowerkit:project:datamodel": {
+        "description": "Commands that operate on the data model"
```

```suggestion
        "description": "Commands that operate on the data model"
```
progject:datamodeldiff?  any suggestions

---

progject:datamodel is correct as it is a stream and we will get series of commands under it.